src/db.rs

```diff
@@ -572,14 +582,20 @@ impl DB {
         DB::open_cf(opts, path, &[])
     }
 
-    /// Open a database with specified options and column family.
-    ///
-    /// A column family must be created first by calling `DB::create_cf`.
+    /// Open a database with the given database options and column family names.
     ///
-    /// # Panics
-    ///
-    /// * Panics if the column family doesn't exist.
+    /// Column families opened using this function will be created with default `Options`.
     pub fn open_cf<P: AsRef<Path>>(opts: &Options, path: P, cfs: &[&str]) -> Result<DB, Error> {
+        let cfs_v = cfs.to_vec().iter().map(|name| ColumnFamilyDescriptor {
```

Comment:
Would it make sense to use `ColumnFamilyDescriptor::new(name, Options::defaut())` instead?